music_assistant/helpers/tags.py

```diff
@@ -64,7 +64,8 @@ def split_artists(
     # when not using the multi artist tag, the artist string may contain
     # multiple artists in freeform, even featuring artists may be included in this
     # string. Try to parse the featuring artists and separate them.
-    splitters = ("featuring", " feat. ", " feat ", "feat.")
+    splitters = ("featuring", " feat. ", " feat ", "feat.", "with", " with ",
+                 " duet with ", "duet with")
```

Comment:
put " duet with" before just "with" to prefer splitting on that first

Reply:
done